### ISSUE TITLE
Filter columns are acting strange on some pages

### ISSUE THREAD

**Issue by lebell** [TNRIS/iswp2017#251]
EX: When I navigate to the usage type page through the main menu different columns appear by default (sometimes 2 sometimes 4), but then if I click one of the hyperlinks and then back in my browser all the columns appear. Prefer to have all pages display with all columns as default.

**Comment by DavidHickman** [comment]
I haven't been able to reproduce this behavior, so we might need to experiment until we can reliably reproduce (so we can verify when it's fixed).

1. Is there a set of selections you make that consistently reproduces this bug?
2. Does it happen more/less in a particular browser?
3. Is this a new bug (staging) or does it happen in production as well?

**Comment by lebell** [comment]
1. Consistently occurring when navigating to the usage type page and then clicking away using a hyperlink on that page and returning using the back button. Usually only missing columns are county and region when page is first opened. 

2. Only occurring for me in chrome. IE seems to work great.

3. The changing of columns based on how you load the page is only happening in staging. Production only displays strategy, source and entity columns.

**Comment by lebell** [comment]
[Laura Bell - Monday, December 04, 2017 12.15.50 PM.zip](https://github.com/TNRIS/iswp2017/files/1528391/Laura.Bell.-.Monday.December.04.2017.12.15.50.PM.zip)

**Comment by DavidHickman** [comment]
Thanks for the update! I can reproduce the behavior you're describing. I'm going to look into it and see if I can determine the cause.

**Comment by lebell** [comment]
The default columns are not changing when navigating from page to page using the hyperlinks anymore- thank you! Could we update the Usage Type page to display all possible columns by default and be the same as other pages? Currently it is not automatically displaying the 'Region' and 'County' columns.

**Comment by DavidHickman** [comment]
Thanks for the feedback, @lebell . I'll take a look.

**Comment by DavidHickman** [comment]
I think I've got this issue resolved, @lebell .

The columns for Usage Type were being actively filtered which must have been a decision made prior to the Fall 2017 updates. I removed the filter and we'll have this comment as documentation that it was removed. The pivot table links should be fixed as well.

As I was testing, I noticed that the columns do not sort on entity pages so that Entity is on the far left. This is the same behavior as in the production version, so I left it unchanged. We can look into updating that behavior if needed.
e.g. https://2017-staging.texasstatewaterplan.org/entity/3110
